Streetcode/Streetcode.WebApi/appsettings.json

```diff
@@ -15,6 +15,41 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "Serilog": {
+    "Using": [],
+    "MinimumLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "System": "Warning"
+    },
+    "Enrich": [
+      "FromLogContext",
+      "WithMachineName",
+      "WithProcessId",
+      "WithThreadId"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console",
+        "Args": {
+          "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss} [{Level:u3}] {Message:lj}{NewLine}{Exception}"
+        }
+      }
+    ],
+    "Filter": [
+      {
+        "Name": "ByExcluding",
+        "Args": {
+          "expression": "EndsWith(RequestPath, '/api/Art/GetAll') or EndsWith(RequestPath, '/api/Audio/GetAll')"
+        }
+      }
+    ],
+    "PropertiesToIgnore": [
+      "id",
+      "base64",
+      "description"
```

Comment:
why do we need to ignore id and description properties? Let's configure this section to ingore what we will need in Prod